add_to_app/books/flutter_module_books/pigeon/schema.dart

```diff
@@ -11,14 +11,12 @@ class Book {
   String summary;
   String publishDate;
   int pageCount;
-  // Thumbnail thumbnail;
+  Thumbnail thumbnail;
 }
 
-// TODO(gaaclarke): add this back when the https://github.com/flutter/flutter/issues/58896
-// crash is resolved.
-// class Thumbnail {
-//   String url;
-// }
+class Thumbnail {
+  String url;
```

Comment:
bonus, if you enable this now, we should fill it with some data in the app :)

Reply:
Good point, it doesn't seem high priority.  If I get a spare sec later this week I can try throwing it in.